lms/djangoapps/student_dashboard/test/view_tests.py

```diff
@@ -0,0 +1,47 @@
+import ddt
+import mock
+import factory
+
+from django.db.models import signals
+from django.test import RequestFactory
+
+from common.lib.nodebb_client.categories import ForumCategory
+from lms.djangoapps.onboarding.tests.factories import UserFactory
+from lms.djangoapps.student_dashboard.views import (get_joined_communities, get_recommended_xmodule_courses,
+                                                    get_enrolled_past_courses)
+from student.models import CourseEnrollment
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
```

Comment:
Please use iSort from shell to sort imports, when I ran it, it shows the following result

```
import ddt
import factory
import mock
from django.db.models import signals
from django.test import RequestFactory

from common.lib.nodebb_client.categories import ForumCategory
from lms.djangoapps.onboarding.tests.factories import UserFactory
from lms.djangoapps.student_dashboard.views import (
    get_enrolled_past_courses,
    get_joined_communities,
    get_recommended_xmodule_courses
)
from student.models import CourseEnrollment
from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
from xmodule.modulestore.tests.factories import CourseFactory
```